skfda/exploratory/stats/_functional_transformers.py

```diff
@@ -342,3 +343,143 @@ def number_up_crossings(
         points_greater & points_smaller_rotated,
         axis=2,
     ).T
+
+
+def unconditional_central_moments(
+    data: Union[FDataBasis, FDataGrid],
+    n: int,
+) -> NDArrayFloat:
+    """
+    Calculate the unconditional central moments of a dataset.
```

Comment:
Give a larger explanation with the formulas, as it is not a widely known concept.